src/v2/Apps/Fair/Components/FairExhibitorRail/FairExhibitorRail.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useRef } from "react"
+import { Box, BoxProps, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { FairExhibitorRail_show } from "v2/__generated__/FairExhibitorRail_show.graphql"
+import { useLazyLoadComponent } from "v2/Utils/Hooks/useLazyLoadComponent"
+import { FairExhibitorRailArtworksQueryRenderer as FairExhibitorRailArtworks } from "./FairExhibitorRailArtworks"
+import { FairExhibitorRailPlaceholder } from "./FairExhibitorRailPlaceholder"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+
+interface FairExhibitorRailProps extends BoxProps {
+  show: FairExhibitorRail_show
+}
+
+/**
+ * Though it is likely to exist, the sale message line may be missing.
+ * In order to avoid the page shifting between the loading state and the ready state,
+ * we need to hardcode the height.
+ */
+export const FAIR_EXHIBITOR_RAIL_HEIGHT = 233
+export const FAIR_EXHIBITOR_IMAGE_HEIGHT = 160
+
+export const FairExhibitorRail: React.FC<FairExhibitorRailProps> = ({
+  show,
+  ...rest
+}) => {
+  const ref = useRef<HTMLDivElement | null>(null)
+  const { isMounted, Waypoint } = useLazyLoadComponent()
```

Comment:
Would it make more sense to update `isMounted` naming to be more specific to lazy loading? We often use `isMounted` in a variety of contexts, from SSR, to client-side only, to accessing sharify, etc. 

If we're specifically using this flag to show content when triggered by the `<Waypoint>` it might increase clarity to tie the names together somehow, which would then point at the `useLazyLoadComponent` hook and create a conceptual relationship.

Reply:
Yeah I dunno; nothing feels right and I kind of just want to install an intersection observer hook. Like, this isn't really a waypoint either because those typically report on their state. This just trips from unmounted => mounted and that's it.